src/webgpu/shader/execution/expression/binary/f32_arithmetic.spec.ts

```diff
@@ -57,11 +49,11 @@ Accuracy: Correctly rounded
       return { input: [f32(lhs), f32(rhs)], expected: f32(f32_lhs + f32_rhs) };
     };
 
-    let cases: Array<Case> = [];
+    const cases: Array<Case> = [];
```

Comment:
Sigh, JavaScript.
Yeah, the 'cases' name always refers to the same object, but that object is mutated over time.

In any case, this new version of the code is cleaner than the old version.